content/post/0-to-60-analytics-stack.md

```diff
@@ -0,0 +1,79 @@
+
+Even though the BI purchase is expensive, it's often the least contentious  since it is the only technology of the three that business people actually see! Build a value proposition by combining expected maintenance costs (in salary/opportunity cost) with the value of avoiding a bad decision due to bad data ($10K?). By my simple back of the envelope calculations, Looker ended up being cheaper than (free) Superset, because it largely runs itself and enables self-serve.
+
+This is one of the purchase decisions that may actually merit a deck or formal document outlining the pros and cons of each option. However, I regret over-democratizing the process by involving multiple business users to help make the decision by committee. At the end of the day, making simple dashboards is similar in each system, and it is you, not business users, who will bear the burden of maintaining permissions, query accuracy, and slack integrations. Reach out to the greater data community and hold yourself accountable to make the right decision.
+
+I made one specific mistake here that I want to share: Looker was much more expensive than the other options, and, to explain the cost, I leaned on the value proposition of teams not needing to hire a dedicated analyst to write SQL queries. This was a mistake for two reasons:
+
+1.  Business users still need a baseline skill level to use Looker. A surprisingly effective rule-of-thumb is that if a person is comfortable with pivot tables in excel, they can do Looker. Do a quick calculation of how many people in your org need data, and of those, how many of them you would trust to make a pivot table.
+
+2. There are benefits to hiring an analyst beyond simply doing SQL queries. Looker saved us from hiring an analyst, but that analyst could also have added value by connecting the dots of cross-functional business requests into product priorities. Unless you are severely budget constrained, it doesn’t hurt to have another bright technical person around, and the earlier you hire them, the better.
+
+Here is a simple chart I wish I would have used to diagnose analytics needs:
+
+![Analyst or Self Service](https://nelsonauner.com/images/analysts_or_self_serve.png)
+
+
+This concludes the three main “courses” of an analytics stack. If you are still hungry, I will touch briefly on two increasingly necessary components as you continue to grow:
+
+## Business Logic Layer:  
+- You might consider: Looker, DBT
+
+ Once you get to more than 3 SQL writers, each will use slightly different JOIN conditions and aggregations, and it will cause problems. These problems may be technical (the same expensive queries getting run repeatedly) but certainly analytical problems as different teams build out slightly different queries for the same metrics.
+Business logic layers solve this problem by centralizing and documenting the (business logic <-> data) interface.
```

Comment:
I'd consider adding another sentence or two explaining what a business logic layer is for folks that have less experience here.